components/bitly/actions/find-bitlink/find-bitlink.mjs

```diff
@@ -0,0 +1,92 @@
+import { ConfigurationError } from "@pipedream/platform";
+import { formatDeepLink } from "../../common/common.utils.mjs";
+import bitly from "../../bitly.app.mjs";
+
+export default {
+  key: "bitly-find-bitlink",
+  name: "Retrieve a Bitlink",
+  description: "Returns information for the specified link.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly,
+    bitlink: {
+      type: "string",
+      description: "Bitlink url",
+    },
+    createBitlinkIfNotFound: {
+      label: "Create a new bitlink if the bitlink is not found?",
+      description:
+        "Create a new bitlink if no record is found for the specified bitlink.",
+      type: "string",
```

Comment:
Maybe we can use boolean here?

Reply:
I think string is still also option considering the fact that this field shows up to the user. [Yes, No] feel good as compared with [true, false]. What do you think?

Reply:
Humm, normally we use boolean in this case :)

Reply:
Okay. I will make the update

Reply:
Done